poker-pals/src/components/Tables.vue

```diff
@@ -1,42 +1,72 @@
 <template>
-    <div id="tables-page">
-        <UserNavbar class="navbar-section"/>
```

Comment:
Is there a reason you removed the nav bar?

Reply:
Must've bugged out while merging, will include it back.